src/Icon/Icon.tsx

```diff
@@ -81,7 +81,7 @@ export const Icon: FC<IconProps> = ({
   color = '',
   size = '',
   icon = '',
-  title,
+  title = 'icon',
```

Comment:
I set a default value for the alt attribute, for accessibility

Reply:
Instead of having a default title, which doesn't actually help understanding what the icon depicts, isn't it better to make title required? I'm not familiar with the codebase, so I may be missing the reason for it being optional.